SeaPkg/Core/Runtime/SeaResponderReport.c

```diff
@@ -367,27 +366,34 @@ SeaResponderReport (
     goto Exit;
   }
 
-  if ((IMAGE_VALIDATION_DATA_HEADER *)(VOID *)AuxFileBase == 0) {
+  AuxFileHdr = (IMAGE_VALIDATION_DATA_HEADER *)(VOID *)AuxFileBase;
+  if (AuxFileHdr == NULL) {
     DEBUG ((DEBUG_ERROR, "%a Reported aux file base address is NULL!\n", __func__));
     Status = EFI_SECURITY_VIOLATION;
     goto Exit;
   }
 
-  if (!IsBufferInsideMmram (AuxFileBase, AuxFileSize)) {
-    DEBUG ((DEBUG_ERROR, "%a Reported aux file (0x%p: 0x%x) does not reside in MMRAM region!\n", __func__, AuxFileBase, AuxFileSize));
+  if (AuxFileHdr->Size > AuxFileSize) {
+    DEBUG ((DEBUG_ERROR, "%a Reported aux file size is larger than the actual size %x %x\n", __func__, AuxFileHdr->Size, AuxFileSize));

```

Comment:
nit: `0x` preceding the hexadecimal value can reduce confusion in interpreting results in logs.

```suggestion
    DEBUG ((DEBUG_ERROR, "%a Reported aux file size is larger than the actual size 0x%x 0x%x\n", __func__, AuxFileHdr->Size, AuxFileSize));
```